packages/themes/package.json

```diff
@@ -64,13 +64,13 @@
 	"type": "module",
 	"exports": {
 		".": {
+			"types": "./dist/index.d.ts",
 			"import": "./dist/index.mjs",
 			"require": "./dist/index.cjs"
 		}
 	},
 	"main": "./dist/index.cjs",
 	"module": "./dist/index.mjs",
-	"types": "./dist/index.d.ts",
```

Comment:
Bitte den Fallback erstmal drin lassen.

Reply:
Ich sehe alle betroffenen package.json wurden bereits aktualisiert. Danke für die Umsetzung!

Ist die PR somit geschlossen?

Reply:
Ich habe aus Versehen Falschrum gemergt und habe das schon rückgängig gemacht. Ich werde mir Ihr Repo holen und den Konflikt lösen und Ihren Commit reinholen. Danach in den main mergen.